x-pack/plugins/monitoring/server/kibana_monitoring/bulk_uploader.ts

```diff
@@ -163,10 +170,50 @@ export class BulkUploader implements IBulkUploader {
     };
   }
 
+  private async getKibanaMetrics() {
```

Comment:
think you'll need to remove changes in this file to get this working now we've removed getMetrics()